src/plotting/create_plots.py

```diff
@@ -123,10 +121,17 @@ def create_plots(args, root_path, use_datetime=True, test_private_keys: Optional
             filename = f"plot-k{args.size}-{plot_id}.plot"
         full_path: Path = args.final_dir / filename
 
-        if args.final_dir.resolve() not in plot_filenames:
-            if str(args.final_dir.resolve()) not in config["harvester"]["plot_directories"]:
+        resolved_final_dir: str = str(Path(args.final_dir).resolve())
+        plot_directories_list: str = config["harvester"]["plot_directories"]
+
+        if args.exclude_final_dir:
+            log.info(f"Not adding directory {resolved_final_dir} to harvester for farming")
+            if resolved_final_dir in plot_directories_list:
+                log.warn(f"Directory {resolved_final_dir} already exists for harvester, you will need to remove it manually")
+        else:
+            if resolved_final_dir not in plot_directories_list:
```

Comment:
Maybe add a log message saying it was added?

Reply:
Agreed adding it.